### ISSUE TITLE
Can't register locally - exported Object VueTransmit is undefined

### ISSUE THREAD

**Issue by MartinMuzatko** [alexsasharegan/vue-transmit#67]
Hello.

I'm trying to implement VueTransmit and found that I can't register it locally.

This is what I'm doing in my SFC:

``` js
import { VueTransmit } from 'vue-transmit';

export default {
    components: { VueTransmit },
    // options and methods truncated...
    created() {
        console.log(VueTransmit)
    },
}
```

This is what I get when I use `<VueTransmit>` then:

```
vue.runtime.esm.js:587 [Vue warn]: Unknown custom element: <VueTransmit> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```

Also, the `console.log` for the imported `{VueTransmit}` object is `undefined`:

![image](https://user-images.githubusercontent.com/2950505/45021021-c7b13b80-b030-11e8-9a08-180eaeecf442.png)

Am I forgetting something?

**Comment by MartinMuzatko** [comment]
Oh! usually you can declare the name like so : `components: {VueTransmit: VueTransmit}`
But this is not the case when you already define a name. Thanks!

**Comment by MartinMuzatko** [comment]
Unfortunately, this does not do the trick for me. I still get ` Unknown custom element: <vue-transmit> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
`

**Comment by alexsasharegan** [comment]
Hmm, that doesn't make sense. The export [here](https://github.com/alexsasharegan/vue-transmit/blob/master/src/index.ts#L36) resolves to this default export of the [vue component here](https://github.com/alexsasharegan/vue-transmit/blob/master/src/components/VueTransmit.vue#L88).

You could always be explicit about it and see what happens:

```html
import { VueTransmit } from "vue-transmit";

const MyComponent = Vue.extend({
  name: "my-component",
  components: {
    "vue-transmit": VueTransmit,
  },
});
```

**Comment by MartinMuzatko** [comment]
I tried in a minimal environment: https://codesandbox.io/s/0mnxl46r8v

**Comment by alexsasharegan** [comment]
Are you using the alpha of v9, or v8? v8 is the stable version, so code sandbox will use that for imports. It doesn't have the modular setup with all the exports.

https://codesandbox.io/s/64p6j1pn6z

https://github.com/alexsasharegan/vue-transmit/tree/8.x

**Comment by alexsasharegan** [comment]
Yeah, alphas aren't considered stable, so they aren't included with the latest tag.

**Comment by alexsasharegan** [comment]
FYI, this is mentioned in [the download section](https://github.com/alexsasharegan/vue-transmit#download).

![image](https://user-images.githubusercontent.com/16269252/45106946-eb39bc00-b0ec-11e8-96c0-9201461b53ef.png)

**Comment by MartinMuzatko** [comment]
I test it with that and then i close the issue. Thank you so much :)

**Comment by alexsasharegan** [comment]
Awesome!

**Comment by alexsasharegan** [comment]
Oh, that totally makes sense! Sorry about that! I hope to get back into the Vue ecosystem soon and devote some time to bringing this up to stable and add a documentation site.
